md/dates-management-tutorial.md

```diff
@@ -0,0 +1,67 @@
+# How to work with dates in forms and BDM
+
+Currently (until versions 7.4.x), only date selection is supported natively in the UI Designer date picker widget. So this page will only focus on date management (not date and time).
+
+## Define your BDM data
+
+Start by defining your business data model.
+
+It is recommended that you define the date attributes of your business data Objects with a **LONG** type instead of DATE if you want the value stored in the database to reflect the date entered by the user. Otherwise the timezone of the server will be used when converting the java date in an SQL date and, unless your server is configured in UTC, an offset will be applied. This will not be an issue if you do not query the business data database directly and only use the API (as the same conversion will be performed the other way around when retrieving the value from the database). In this case, you can safely use a DATE attribute type.
+
+![BDM](images/dates-management-tutorial/tuto-dates-bdm.png) <!--{.img-responsive .img-thumbnail}-->
+
+Then, in the process, create a new business variable of the type of you business object.
+
+![Business variable](images/dates-management-tutorial/tuto-dates-variable.png) <!--{.img-responsive .img-thumbnail}-->
+
+## Define the contract and task operations or business variable initialization script
+
+Now you need to define the *process instantiation* or *task execution* contract (depending on your use case).  
+The date picker widget output is an ISO 8601 formatted String ("2017-01-13T00:00:00.000Z") that will be automatically formatted to a java.util.Date by process instantiation or task execution REST API if the contract input is of type **DATE**. The date/time received will be considered as UTC. The time is set to midnight by the widget as it doesn't handle time.  
```

Comment:
"Will be considered" as the contract supports formats that do not include the "z" like "2017-01-13T00:00:00". So you can pass a date that is not a UTC date, but it will be considered as UTC when it is converted to a Date object.
Maybe I should say "will be considered as UTC when it is converted to a Date object"